src/qualitycontrol.jl

```diff
@@ -89,7 +105,7 @@ end
 """
```

Comment:
`showall!` is a bit tricky, as Base used to have a `showall` function, which is still in e.g. DataFrames. That function is an override of `show(..., compact = false)`. Maybe `unfilter!` is better?

Reply:
I also briefly considered `reset!` or `unmask!`, because of the potential name confusion with `showall` (which does a different thing anyways). The alternative would be to use a special function, like `filter!(all_occurrences, occurrences)`, but this seems even more counter-intuitive.

Reply:
I like `unfilter!`, or if you want to stay in the same logic maybe `allrecords!`?

Reply:
I like `allrecords!` -- it's simple and descriptive.